lib/src/components/NxDrawer/NxDrawer.scss

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use '../../scss-shared/nx-container-helpers';
+ @use '../../scss-shared/nx-text-helpers';
+ 
+$-box-shadow-offset: 52px;
+
+.nx-drawer {
+  // Undo styles that chrome applies to <dialog> elements
+  border: initial;
+  height: initial;
+  left: initial;
+  margin: initial;
+  max-height: initial;
+  max-width: initial;
+  padding: initial;
+  width: initial;
+
+  // Prevent brief flash of scrollbar as tooltips disappear
+  overflow: hidden;
+
+  &::backdrop {
+    // Chrome tries to add its own subtle mask here which doubles up with our own
+    display: none;
+    pointer-events: auto;
+  }
+
+  background: transparent;
+  box-sizing: border-box;
+  height: 100%;
+  padding-left: $-box-shadow-offset;
+  position: fixed;
+  right: 0;
+  top: 0;
+  transform: translateX(100%);
+  width: 520px + $-box-shadow-offset;
+  z-index: 100;
+
+  &--with-global-header {
+    $global-header-height: 43px;
+    height: calc(100% - #{$global-header-height});
+    top: $global-header-height;
+  }
+  
+  &--narrow {
+    width: 348px + $-box-shadow-offset;
+  }
+
+  &__inner {
+    align-items: stretch;
+    background-color: var(--nx-swatch-white);
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    padding: var(--nx-spacing-6x);
+    position: relative;
+
+    &::after {
+      content: " ";
+
+      box-shadow: -2px 0px 50px rgba(128, 128, 128, 0.5);      
+      height: calc(100% - var(--nx-spacing-16x));
+      width: 100%;      
```

Comment:
There's trailing spaces on many lines in this file.  I recommend setting up your editor so that they are visible.

Reply:
> There's trailing spaces on many lines in this file. I recommend setting up your editor so that they are visible.

Fixed trailing spaces here: c9f583538c2c5d9a92f55163cab5cf14db39d55a

Reply:
> Fixed trailing spaces here: https://github.com/sonatype/sonatype-react-shared-components/commit/c9f583538c2c5d9a92f55163cab5cf14db39d55a

There still are some/are some again

Reply:
> > Fixed trailing spaces here: [c9f5835](https://github.com/sonatype/sonatype-react-shared-components/commit/c9f583538c2c5d9a92f55163cab5cf14db39d55a)
> 
> There still are some/are some again

I think I got them all this time: 30b14c34fde1c9fe6095372351f6062969a3bbc8